### PR TITLE
updating key.properties script in yaml

### PR DESCRIPTION
**Pull request by icarusdust**

### CODE REVIEW

**Comment by icarusdust** [comment]
Fixes #5

File: integrations/lambdatest_integration_demo_project/lambdatest_integration_demo_project/android/app/build.gradle [hunk 1]
```diff
@@ -146,11 +146,18 @@ android {
...
+                storeFile file(System.getenv()["CM_KEYSTORE_PATH"])
+                storePassword System.getenv()["CM_KEYSTORE_PASSWORD"]
+                keyAlias System.getenv()["CM_KEY_ALIAS_USERNAME"]
+                keyPassword System.getenv()["CM_KEY_ALIAS_PASSWORD"]
+            } else {
+                storeFile file("/path/to/local/myreleasekey.keystore")
+                storePassword "password"
+                keyAlias "MyReleaseKey"
+                keyPassword "password"
+            }
```
Comment (nevercode-kevin) [inline integrations/lambdatest_integration_demo_project/lambdatest_integration_demo_project/android/app/build.gradle:160]: I don't think we should encourage people to expose passwords that would be committed to source control. Maybe use a similar syntax to show reading the values from the local machine environment instead.

Comment (icarusdust) [reply integrations/lambdatest_integration_demo_project/lambdatest_integration_demo_project/android/app/build.gradle:160]: Well, I do not think that else condition is used anyways. We can get rid of it as well.

Comment (nevercode-kevin) [reply integrations/lambdatest_integration_demo_project/lambdatest_integration_demo_project/android/app/build.gradle:160]: What if somebody wants to run a build on their local machine?

Comment (icarusdust) [reply integrations/lambdatest_integration_demo_project/lambdatest_integration_demo_project/android/app/build.gradle:160]: I added a comment on this inside the yaml file. If they want to configure their code signing differently, then key.properties way of doing it is possible, I guess. I added a link to the docs as well.

Comment (icarusdust) [reply integrations/lambdatest_integration_demo_project/lambdatest_integration_demo_project/android/app/build.gradle]: Added the changes.

**Pull request merged**